RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1279,7 +1291,225 @@ private void parseMediaPlayerMRUList(String regFileName, AbstractFile regFile, B
                 line = line.trim();
             }
         }
-        if (bbartifacts != null) {
+        if (!bbartifacts.isEmpty()) {
+            postArtifacts(bbartifacts);
+        }
+    }
+    
+     /**
+     * Create recently used artifacts to parse the regripper output
+     * 
+     * @param regFileName name of the regripper output file
+     * 
+     * @param regFile registry file the artifact is associated with
+     * 
+     * @param reader buffered reader to parse adobemru records
+     * 
+     * @throws FileNotFound and IOException
+     */
+    private void parseGenericMRUList(String regFileName, AbstractFile regFile, BufferedReader reader) throws FileNotFoundException, IOException {
```

Comment:
The regFileName argument of all of these methods seems to be unused now, and could be removed.

Reply:
Removed